platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFile.java

```diff
@@ -56,4 +62,49 @@ public Path getConfigFilePath() {
     }
 
     public abstract void createConfig() throws ConfigurationFileException;
+
```

Comment:
When does this outputstream get closed? `ConfigurationHandler.write()` is javadoc'd as not closing the outputstream. `ConfigurationHandler` is called by the Felix implementation of `PersistenceStrategy`. Am I missing an implementation of our own?